pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -444,6 +444,10 @@ def apply_operations(self, operations):
             elif isinstance(operations[0], BasisState):
                 self._apply_basis_state(operations[0].parameters[0], operations[0].wires)
                 operations = operations[1:]
+            elif isinstance(operations[0], MPSPrep):
+                mps = operations[0].mps
+                self._tensornet.updateMPSSitesData(mps)
```

Comment:
Please add unit tests for this new feature and error message should be thrown out if the method is `tn`

Reply:
Done!